### PR TITLE
ci(action): auto-bump daily and change slack notification channel

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Change the slack channel 
Run daily

## Why is it important?

There will be only 24 hours daily between the golang-crossbuild auto-bumps and this one.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `./changelog/fragments` using the [changelog tool](https://github.com/elastic/elastic-agent#changelog)
- [ ] I have added an integration test or an E2E test

## Author's Checklist

- [ ]

## Related issues

Similar to https://github.com/elastic/golang-crossbuild/pull/367

## Questions to ask yourself

- How are we going to support this in production? 
- How are we going to measure its adoption? 
- How are we going to debug this?
- What are the metrics I should take care of?
- ...

### CODE REVIEW

File: .github/workflows/bump-golang.yml [hunk 1]
```diff
@@ -21,3 +24,5 @@ jobs:
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
           pipeline: ./.github/updatecli-bump-golang.yml
+          notifySlackChannel: "#ingest-notifications"
+          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, `@update-me-with-the-slack-team-to-be-poked` please look what's going on <${{ env.JOB_URL }}|here>"
```
Comment (v1v) [inline .github/workflows/bump-golang.yml:28]: What team should be pinged? 
```suggestion
          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, `@UPDATEME` please look what's going on <${{ env.JOB_URL }}|here>"
```

Comment (cmacknz) [reply .github/workflows/bump-golang.yml:28]: #ingest-notifications

Comment (v1v) [reply .github/workflows/bump-golang.yml:28]: That's the channel but within the channel, it's possible to ping a given team with the `@team`. That's quite handy when there too many notifications in the slack channel

**Comment by v1v** [comment]
CI error unrelated to this change, it failed when processing the test results

image

File: .github/workflows/bump-golang.yml (hunk 1 above)
Comment (v1v) [reply .github/workflows/bump-golang.yml:28]: For instance, currently,  we are the ones receiving those notifications in Slack:

image

**Review by pierrehilbert — APPROVED** [review]

File: .github/workflows/bump-golang.yml (hunk 1 above)
Comment (pierrehilbert) [reply .github/workflows/bump-golang.yml:28]: @agent-team shoud do the job

**Comment by pierrehilbert** [comment]
I fixed the processing step

**Comment by v1v** [comment]
The CI failures are unrelated to the nature of this PR. 

image

**Comment by pierrehilbert** [comment]
https://github.com/elastic/elastic-agent/pull/3901 will fix the test error

**Comment by v1v** [comment]
I have no clue what's the reason for the failure in the CI, but this PR does not change anything that the CI can test/verify. It changes the automation in GitHub actions.

@pierrehilbert, how should we proceed?

**Comment by pierrehilbert** [comment]
This error is a new one, I'm looking at it to report it and if this can't be related to the change, will force merge this one.

**Pull request merged**